monitoring/front_end_exporter.py

```diff
@@ -38,6 +44,10 @@ def multi_dict_unpacking(lst):
 
 
 def parse_metric(metrics):
+    if not len(metrics):
```

Comment:
Just for the sake of being explicit, can you change this to `if len(metrics) == 0:`